chef/cookbooks/crowbar/files/default/crowbar.service

```diff
@@ -8,7 +8,7 @@ User=crowbar
 Group=crowbar
 PIDFile=/var/run/crowbar/crowbar.pid
 ExecStartPre=/usr/bin/rm -f /var/run/crowbar/looper-chef-client.lock /var/run/crowbar/chef-client.lock
-ExecStart=/usr/bin/rainbows -E production -c /opt/dell/crowbar_framework/rainbows.cfg
+ExecStart=/usr/bin/puma.ruby2.1 -C /opt/dell/crowbar_framework/config/puma.rb
```

Comment:
I think we don't need the `.ruby2.1` suffix. For HAWK it works fine without that.